agent/status.go

```diff
@@ -107,11 +107,9 @@ func (a *Agent) statusHandler() http.HandlerFunc {
 }
 
 type triggerEndTaskResp struct {
-	Description string `json:"desc,omitempty"`
-	Status      string `json:"status,omitempty"`
-	Type        string `json:"type,omitempty"`
-	// TODO (DEVPROD-5122): add documentation once the additional features for
```

Comment:
is this done or is documentation still needed? Not sure what this is referring to.

Reply:
It's already documented, so it can be removed.